script/libtest.bash

```diff
@@ -231,10 +232,7 @@ test_must_fail_without_rbac_roles() {
     echo_info "RBAC TEST: function deploy/call must fail without RBAC roles"
     _delete_simple_function
     kubeless_recreate $KUBELESS_MANIFEST_RBAC $KUBELESS_MANIFEST
-    _wait_for_kubeless_controller_ready
-    _deploy_simple_function
-    _wait_for_kubeless_controller_logline "User.*cannot"
-    _call_simple_function 1
```

Comment:
why did you remove this?

Reply:
I removed this because without RBAC the controller would not be able to read the config file. So it would fail even before with the same error.